vnext/Microsoft.ReactNative.ComponentTests/Microsoft.ReactNative.ComponentTests.vcxproj

```diff
@@ -105,6 +105,8 @@
       </AdditionalIncludeDirectories>
       <UseFullPaths>true</UseFullPaths>
       <CallingConvention>Cdecl</CallingConvention>
+      <!-- RuntimeTypeInfo is required by JSI -->
+      <RuntimeTypeInfo>true</RuntimeTypeInfo>
```

Comment:
Did your change introduce that, or was that always the case?

Reply:
It is required by JSI. The MS.RN already uses RTTI. This adds the same to the RN.dll. We need to change the JSI to avoid RTTI.

---
In reply to: [470186379](https://github.com/microsoft/react-native-windows/pull/5710#discussion_r470186379) [](ancestors = 470186379)

Reply:
I don't see MS.RN using RTTI via RuntimeTypeInfo, do you know where this is set?

---
In reply to: [470243854](https://github.com/microsoft/react-native-windows/pull/5710#discussion_r470243854) [](ancestors = 470243854,470186379)

Reply:
You are right: MS.RN explicitly disables RTTI. But, JSI needs it... I think it is an unpleasant JSI bug. Let me see how I can reduce the scope of RTTI here.

---
In reply to: [470472946](https://github.com/microsoft/react-native-windows/pull/5710#discussion_r470472946) [](ancestors = 470472946,470243854,470186379)